ide-common/src/main/kotlin/org/digma/intellij/plugin/notifications/InsightsNotificationStartupActivity.kt

```diff
@@ -0,0 +1,11 @@
+package org.digma.intellij.plugin.notifications
+
+import com.intellij.openapi.components.service
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.startup.StartupActivity
+
+class InsightsNotificationStartupActivity: StartupActivity {
+    override fun runActivity(project: Project) {
+        project.service<InsightsNotificationsService>().waitForEvents()
```

Comment:
the name `waitForEvents` sounds blocking. maybe just name it `startWorking`

Reply:
I'll keep it, its a background thread